### PR TITLE
feat(tui): add Spanish (`es`) locale

### PR DESCRIPTION
**Pull request by fujiapple852**

### CODE REVIEW

**Comment by josueBarretogit** [comment]
Can I be the reviewer?

**Comment by fujiapple852** [comment]
That would be great @josueBarretogit, thank you!

**Review by josueBarretogit — COMMENTED** [review]
These changes will make it more natural

File: crates/trippy-tui/locales/app.yml [hunk 1]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "设置"
+  es: "Configuraciones"
 bsod_failed:
   en: "Trippy Failed :("
   fr: "Trippy a échoué :("
   tr: "Trippy Başarısız :("
   it: "Trippy ha avuto un problema :("
   pt: "Trippy falhou :("
   zh: "Trippy 失败 :("
+  es: "Trippy Falló :("
```
[the remark this answers is not in the document]
Comment (fujiapple852) [reply crates/trippy-tui/locales/app.yml:391]: I've updated it to "Trippy tuvo un problema :("

**Review by josueBarretogit — APPROVED** [review]
Looks Good To Me, Ñ

**Pull request merged**